src/main/assets/scss/main.scss

```diff
@@ -252,7 +252,7 @@ $callout-bg: #f0f4f5;
 @media (max-width: 640px) {
   .parent-box {
     .floating-back-to-top {
```

Comment:
Is this still needed now that it's fixed to the bottom left?

Reply:
removed and update the code.